backend/src/test/resources/graphql-test/update-aoe-questions.graphql

```diff
@@ -0,0 +1,19 @@
+mutation UpdateAOE(
```

Comment:
did we also want to add `testResultDelivery` here? I know we aren't using this quite yet in our tests